armi/physics/neutronics/latticePhysics/latticePhysicsInterface.py

```diff
@@ -224,7 +228,7 @@ def interactCoupled(self, iteration):
         iteration : int
             This is unused since cross sections are generated on a per-cycle basis.
         """
-        if self.r.p.timeNode == 0:
+        if self.cs[CONF_RUN_TYPE] == "Snapshots" or self.r.p.timeNode == 0:
```

Comment:
Add a comment describing why this is nesseasry and the logic

something like snapshots may be lower power and different temperature destruction, so it must run in coupled since the interactBOC call runs prior to TH temperature updates.

Reply:
Addressed in https://github.com/terrapower/armi/pull/1206/commits/65edad525798b462c55fc0ce7d1bf7bb10fdb1b9. Let me know what you think?

Reply:
comment above the line of code would be more helpful in my opinion.

Reply:
https://github.com/terrapower/armi/pull/1206/commits/66eed254772a3cea047f9a29d9973becfe0a487c